docs/index.md

```diff
@@ -0,0 +1,49 @@
+
+MetalLB has support for local traffic, meaning that the machine that receives the data will be the machine that services the request. It is not suggested to use a virtual IP with high traffic workloads because only one machine will receive the traffic for a service - the other machines are solely used for failover.
+
+BGP does not have this limitation but does see nodes as the atomic unit. This means if the service is running on two of five nodes then only those two nodes will receive traffic, but they will each receive 50% of the traffic even if one of the nodes has three pods and the other only has one pod running on it. It is recommended to use node anti-affinity to prevent Kubernetes pods from stacking on a single node.
+
+[note type="important" status="Note"]
+For more information on configuring MetalLB with Calico in BGP mode, please see this [explanation of the required configuration](https://metallb.universe.tf/configuration/calico/) from the [MetalLB website](https://metallb.universe.tf/)
+[/note]
+
+
+## In this documentation
+
+|  |  |
+|--|--|
+| [Tutorials](/t/charmed-metallb-tutorial-overview/11359?channel=1.28/stable)</br>  Get started - a hands-on introduction to using Charmed Metallb operator for new users </br> |  [How-to guides](/t/charmed-metallb-how-to-managed-units/11363?channel=1.28/stable) </br> Step-by-step guides covering key operations and common tasks |
+| [Reference](https://charmhub.io/metallb/actions?channel=1.28/stable) </br> Technical information - specifications, APIs, architecture | [Explanation](/t/charmed-metallb-explanation/####?channel=1.28/stable) </br> Concepts - discussion and clarification of key topics  |
+
+
+# Navigation
+
+| Level | Path                 | NavLink                                                                          |
+|-------|----------------------|----------------------------------------------------------------------------------|
+| 1     | tutorial             | [Tutorial]()                                                                     |
+| 2     | t-overview           | [1. Introduction](/t/charmed-metallb-tutorial-overview/11359)                    |
+| 2     | t-setup-environment  | [2. Set up the environment](/t/charmed-metallb-tutorial-setup-environment/11360) |
+| 2     | t-deploy-metallb     | [3. Deploy MetalLB](/t/charmed-metallb-tutorial-deploy-metallb/11361)            |
+| 2     | t-configure          | [4. Configure MetalLB](/t/charmed-metallb-tutorial-configure/11362)              |
+| 1     | how-to               | [How To]()                                                                       |
+| 2     | how-to               | [1. How To - Managed Units](/t/charmed-metallb-how-to-managed-units/11363)       |
```

Comment:
i think this path is wrong -- it should probably be `h-managed-units`.  and while you're here, let's not repeat `How To` in the navlink verbiage.